presto-docs/src/main/sphinx/release/release-0.173.rst

```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.173
+=============
+
+General Changes
+---------------
+
+* Fix issue where selective aggregates for ``count`` with a constant argument were being ignored.
```

Comment:
The phrase "selective aggregates" isn't very understandable to the end user. Let's say
```
* Fix issue where ``FILTER`` was ignored for :func:`count` with a constant argument.